com.katawa_shoujo.KatawaShoujo.appdata.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>com.katawa_shoujo.KatawaShoujo</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>CC-BY-NC-SA-3.0</project_license>
+	<name>Katawa Shoujo</name>
+	<summary>Katawa Shoujo is a bishoujo-style visual novel set in the fictional Yamaku High School for disabled children, located somewhere in modern Japan.</summary>
+
+	<description>
+		<p>
+			Katawa Shoujo is a bishoujo-style visual novel set in the fictional Yamaku High School for disabled children, located somewhere in modern Japan. Hisao Nakai, a normal boy living a normal life, has his life turned upside down when a congenital heart defect forces him to move to a new school after a long hospitalization. Despite his difficulties, Hisao is able to find friends—and perhaps love, if he plays his cards right. There are five main paths corresponding to the 5 main female characters, each path following the storyline pertaining to that character.
+		</p>
+		<p>
+			The story is told through the perspective of the main character, using a first person narrative. The game uses a traditional text and sprite-based visual novel model with an ADV text box. 
+		</p>
+		<p>
+			Katawa Shoujo contains adult material, and was created using the Ren'Py scripting system. It is the product of an international team of amateur developers, and is available free of charge under the Creative Commons BY-NC-ND License.
```

Comment:
isn't this BY-NC-SA? Based on the licence code above.

Reply:
It probably is, yes :)